### PR TITLE
Docker image fixes

### PR DESCRIPTION
**Pull request by calind**
It mainly fixes the complaint that it cannot find `templates/layout` and some config defaults.

BTW. do you consider publishing images on docker hub?

### CODE REVIEW

File: docker/entrypoint.sh [hunk 1]
```diff
@@ -4,8 +4,8 @@ cat <<EOF > /etc/orchestrator.conf.json
 {
   "Debug": true,
   "ListenAddress": ":3000",
-  "MySQLTopologyUser": "${ORC_TOPOLOGY_USER:-:orchestrator}",
-  "MySQLTopologyPassword": "${ORC_TOPOLOGY_PASSWORD:-:orchestrator}",
```
Comment (shlomi-noach) [inline docker/entrypoint.sh:8]: why this change?

File: Dockerfile [hunk 2]
```diff
@@ -15,11 +15,12 @@ RUN set -ex \
     && { go get -d github.com/github/orchestrator ; : ; } \
     && cd $GOPATH/src/github.com/github/orchestrator \
     && bash build.sh -b \
-    && cp ./docker/entrypoint.sh /entrypoint.sh \
```
Comment (shlomi-noach) [inline Dockerfile:18]: explain the removal of this line?

File: docker/entrypoint.sh (hunk 1 above)
Comment (calind) [reply docker/entrypoint.sh:8]: Because it would default to 
```
"MySQLTopologyUser": ":orchestrator",
"MySQLTopologyPassword": ":orchestrator"
```
instead of
```
"MySQLTopologyUser": "orchestrator",
"MySQLTopologyPassword": "orchestrator"
```
if `ORC_TOPOLOGY_USER` or `ORC_TOPOLOGY_PASSWORD` is not set (Notice the extra `:` in the original version).

File: Dockerfile (hunk 2 above)
Comment (calind) [reply Dockerfile:18]: The entrypoint is added later in the `Dockerfile` (`ADD docker/entrypoint.sh /entrypoint.sh`) so that if it's changed `docker build` picks up the change and rebuild the the container.

File: docker/entrypoint.sh (hunk 1 above)
Comment (shlomi-noach) [reply docker/entrypoint.sh:8]: Can you explain why `:orchestrator` is preferable to `orchestrator` as default username/password?

**Comment by shlomi-noach** [comment]
> do you consider publishing images on docker hub?

Not really. I'm not very familiar with docker, and all docket file in this repo are community contributions. I'm unsure what it would take to publish images on docker hub.

File: docker/entrypoint.sh (hunk 1 above)
Comment (calind) [reply docker/entrypoint.sh:8]: It's the other way around. The current version defaults to `:orchestrator`. This PR changes this to `orchestrator`.

Comment (shlomi-noach) [reply docker/entrypoint.sh:8]: Oh! You are right, of course. I'm not sure if there was an intention behind `:orchestrator`. Looking at blame/history but I don't find anything that suggests the `:` is of importance. 
Since I'm not familiar with docker, dumb question: is this perhaps a template variable syntax or similar?

Comment (calind) [reply docker/entrypoint.sh:8]: That's actaully shell code, which generates a sample config if there is no config file already present at `/etc/orchestrator.conf.json`. The bug was there since my initial commit, I just found it now.

**Review by shlomi-noach — APPROVED** [review]
thank you!
